### DISCUSSION TITLE
Does Quarkus 3.2 LTS support (or will support) Java 21?

### DISCUSSION THREAD

**Discussion by jorsol** [quarkusio/quarkus#36886] [Q&A]
Since Quarkus 3.2 is the first of the  [Long Term Support releases of Quarkus](https://quarkus.io/blog/lts-releases/), and now that Quarkus officially supports Java 21 in 3.5, is there any plan to certify Quarkus 3.2 LTS in Java 21? Or we should wait until the second LTS?

This might be a feature request and might not fit in the scope of an LTS release, but just want to be sure if there is any plan to support Java 21 or not.

**Comment by toby1984** [comment]
Good question! We just started with a new greenfield project based on JDK 21 & Quarkus (currently on 3.5.1) and this is of great interest to us as well.
